addons/mail/static/src/core/public_web/discuss_client_action.js

```diff
@@ -37,6 +37,7 @@ export class DiscussClientAction extends Component {
             props.action.context.active_id ??
             props.action.params?.active_id ??
             this.store.Thread.localIdToActiveId(this.store.discuss.thread?.localId) ??
+            props.action.params?.discuss_active_id ??
```

Comment:
What's the difference between `params.active_id` and this new `discuss_active_id`?
They look the same so why not just reuse `params.active_id`?

Reply:
`active_id` breaks the url. That's why we need to pass the `discuss_active_id` separately so we can handle it our way. rather than letting `web` handle it.